migrations/0001_initial.py

```diff
@@ -18,13 +19,16 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
                 ('user_title', models.CharField(help_text=b'Title to assign after analysis is generated.', max_length=255, null=True, verbose_name=b'User defined title for analysis', blank=True)),
+                ('user_extent', models.CharField(help_text=b'User defined BBOX for analysis extent', max_length=255, null=True, verbose_name=b'Analysis extent', blank=True)),
```

Comment:
All this changes should be included in the next migration --> 003.
Migration file is like a record that keeps the structure version of database. If you migrate staging like this, django will expect that start_time is already exists in 001, in fact, it doesn't exists in 001 in that current database.